lib/services/samples-service.ts

```diff
@@ -218,5 +219,26 @@ export class SamplesService implements ISamplesService {
 		let matchedCategory = _.find(sortedCategories, category => category.regEx.test(description));
 		return matchedCategory ? matchedCategory.id : null;
 	}
+	
+	private getGitHubAccessToken(queryToken: string): IFuture<string> {
```

Comment:
This function is badly named. It does not return the token. It returns a part of an url, containing the token.